test/api/controllers/bulkController.test.js

```diff
@@ -286,4 +290,80 @@ describe('Test the bulk controller', () => {
       should(response.deleted).be.eql(2);
     });
   });
+
+  describe('#updateByQuery', () => {
+    let query;
+    let changes;
+    let esResponse;
+
+    beforeEach(() => {
+      query = {
+        match: { foo: 'bar' }
+      };
+      changes = {
+        bar: 'foo'
+      };
+
+      request.input.body = { query, changes };
+
+      esResponse = {
+        updated: 1
+      };
+
+      kuzzle.ask
+        .withArgs('core:storage:public:bulk:updateByQuery')
+        .resolves(esResponse);
+    });
+
+    it('should forward to the store module', async () => {
+      request.input.args.refresh = 'wait_for';
+      const response = await controller.updateByQuery(request);
+
+      should(kuzzle.ask).be.calledWith(
+        'core:storage:public:bulk:updateByQuery',
+        index,
+        collection,
+        query,
+        changes,
+        { refresh: 'wait_for'});
+
+      should(response).be.eql(esResponse);
+    });
+
+    it('should have default value for refresh param', async () => {
+      await controller.updateByQuery(request);
+
+      should(kuzzle.ask).be.calledWith(
+        'core:storage:public:bulk:updateByQuery',
+        index,
+        collection,
+        query,
+        changes,
+        { refresh: 'false'});
+    });
+
+    it('should throw an error if body.query is malformed', async () => {
+      request.input.body = { query: 'not an object', changes };
+
+      return should(controller.updateByQuery(request)).be.rejectedWith(
```

Comment:
Could be awaited instead or you should remove the `async` since you don't use await in this function